package.json

```diff
@@ -29,15 +30,20 @@
   "homepage": "https://github.com/hoodiehq/hoodie-server-store#readme",
   "devDependencies": {
     "coveralls": "^2.11.6",
+    "memdown": "^1.1.2",
     "nock": "^7.0.1",
     "nyc": "^5.3.0",
+    "pouchdb": "^5.2.1",
+    "request": "^2.69.0",
     "semantic-release": "^6.0.3",
     "standard": "^6.0.4",
     "tap": "^5.0.1"
   },
   "dependencies": {
+    "boom": "^3.1.2",
+    "express": "^4.13.4",
+    "express-pouchdb": "^1.0.2",
     "h2o2": "^5.0.0",
-    "hapi": "^13.0.0",
-    "joi": "^8.0.1"
+    "hapi": "^13.0.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
we can add `"tonicExampleFilename": "example.js"` if you like, see https://tonicdev.uservoice.com/knowledgebase/articles/765846-how-do-i-customize-the-example-for-my-npm-package